pages/library/viewBook.js

```diff
@@ -0,0 +1,136 @@
+import Link from "next/link";
+import Image from "next/image";
+import styled from "styled-components";
+
+const BookPageStyle = styled.ul`
+  position: relative;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  color: black;
+  list-style: none;
+
+  padding-left: 0;
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  margin-left: 3rem;
+  margin-right: 3rem;
+
+  li {
+    position: relative;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  }
+
+  ::first-letter {
+    font-size: 3rem;
+    font-weight: bold;
+  }
+`;
+
+const BookTitle = styled.h1`
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  font-family: "Franklin Gothic Medium";
+  color: black;
+  font-size: 3.5rem;
+`;
+
+const BookPageContainer = styled.div`
+  height: calc(100vh - 8rem);
+  margin-top: 3rem;
+  overflow: scroll;
+`;
+
+const FeatherLink = styled(Link)`
+  position: fixed;
+  bottom: 0;
+`;
+
+const StyledWisdom = styled.li`
+  position: relative;
+
+  li {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+`;
+
+const BookPageImage = styled.div`
+  position: relative;
+  z-index: -2;
+  height: 100%;
+
+  &::before {
+    content: "";
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-image: url("/assets/bookpage.png");
+
+    background-size: cover;
+    opacity: 0.7;
+    z-index: -1;
+  }
+`;
+
+export default function ViewBook({ library, currentBook }) {
+  const categories = library
+    .filter((entry) => entry.book === currentBook)
+    .map((wisdom) => wisdom.category)
+    .filter((value, index, self) => self.indexOf(value) === index);
+
+  return (
+    <>
+      <BookPageContainer>
+        <BookPageImage>
+          <BookTitle>
+            <p>____________</p>
```

Comment:
Using `p`-elements for breaks is not a good practice as it can lead to accessibility issues and can also make the HTML structure less semantic. I think a `hr` or a `<div className="separator" />` would be the better option here.